UltraStar Play/Assets/Scenes/Sing/SingSceneController.cs

```diff
@@ -108,7 +108,15 @@ void Start()
             {
                 playerProfilesWithoutMic.Add(playerProfile);
             }
-            CreatePlayerController(playerProfile, micProfile);
+            PlayerController playerController = CreatePlayerController(playerProfile, micProfile);
+
+            if (SceneData.PlayerProfileToScoreDataMap.Count > 0)
+            {
+                if (SceneData.PlayerProfileToScoreDataMap.TryGetValue(playerProfile, out PlayerScoreControllerData scoreData))
```

Comment:
any reason for those two if's not being one if?

Reply:
Nope. The first if is just useless. I guess at some point, there was more in this if-branch.